### ISSUE TITLE
Deinonychus cannot pounce other creatures

### ISSUE THREAD

**Issue by dinoman9877** [FossilsArcheologyRevival/FossilsArcheologyRevival#1883]
I've tried this with several vanilla and F&A mobs, and thus far, regardless of size, the result is almost always the same.

The Deinonychus will bite a mob a few times, usually two or three. In the case of mobs like the cow or sheep, this is enough to kill it outright and thus they never try to use RPR on them. For any mobs with health greater than that, they will then begin to try using their pounce to latch on.

This has only worked one time so far, against a baby zombie. Other attempts against baby zombies have been unsuccessful, however, meaning it was more luck or some fluke than the pounce actually working correctly.

In literally all other cases so far, when the Deinonychus attempts to pounce its intended target, it simply...bounces off, oftentimes the bounce takes it straight upward.

Further testing had them occasionally pouncing successfully on Pachycephalosaurus and even once on an Allosaurus, though both times had large mobs of the raptors attempting to pounce several times and failing before there was a success. In cases of smaller groups or even individuals attempting to pounce and failing, the results were naturally fatal.

Here they can be seen attempting to pounce zombies, and launching into the air upon failure.
![2021-09-12_22 10 30](https://user-images.githubusercontent.com/68574459/133014629-5e0967ea-ac07-4f09-bf4c-1e54c0f74cfd.png)

The same with a Pachycephalosaurus.
![2021-09-12_22 21 23](https://user-images.githubusercontent.com/68574459/133014676-de7e29ff-b7d6-4c3a-8b17-92bf0c1c4905.png)

And a villager.
![2021-09-12_22 03 41](https://user-images.githubusercontent.com/68574459/133014704-b7a22eb5-4aa5-49ee-84b1-b7ccb7ec84bc.png)

And a Megaloceros. It's obviously making contact here, but the game just doesn't latch. I had to use a female because the male was not such a fan of the Deinonychus' lead in biting.
![2021-09-12_22 06 04](https://user-images.githubusercontent.com/68574459/133014737-4b9cc25a-8d3b-4c17-b604-573e5bea918e.png)

The largest animal they seem to consider prey is the Platybelodon, which was successfully pounced only one. All other tests ended the same as above.
![2021-09-12_22 31 48](https://user-images.githubusercontent.com/68574459/133015321-c8a97451-ca5f-4389-b53e-54623c390a7a.png)

So in short, the Deinonychus largely seem unable to pounce anything unless spawned directly on top of the prey item, and even then it's rather hit or miss. Just wanted this issue to be known.
